Gems/ROS2/Code/Include/ROS2/ROS2Bus.h

```diff
@@ -50,6 +50,10 @@ namespace ROS2
         //! @note Transforms are already published by each ROS2FrameComponent.
         //! Use this function directly only when default behavior of ROS2FrameComponent is not sufficient.
         virtual void BroadcastTransform(const geometry_msgs::msg::TransformStamped& t, bool isDynamic) const = 0;
+
+        //! Get expected simulation loop time in seconds.
+        //! Returns expected loop execution time computed from statics on past frames.
```

Comment:
Did you mean `statistics`?
```suggestion
        //! Returns expected loop execution time computed from statistics on past frames.
```